src/groupeddataframe/groupeddataframe.jl

```diff
@@ -716,7 +716,7 @@ julia> k = keys(gd)[1]
 GroupKey: (a = :foo, b = 2)
 
 julia> keys(k)
-2-element Array{Symbol,1}:
+2-element Array{Symbol, 1}:
```

Comment:
This is the actual output:
```suggestion
2-element Array{Symbol,1}:
```

Reply:
ah - right! We should not change the docstrings.

Reply:
Whoops! Yeah, I was trying to avoid those, good catch!